IceCreamShop.playground/Contents.swift

```diff
@@ -1 +1,98 @@
+import UIKit
+
+struct Flavor{
+    var name: String
+    var rating: Int
+}
+
+enum Size: Double{
+    case small = 3.99
+    case medium = 4.99
+    case large = 5.99
+}
+
+struct Cone{
+    let flavor: Flavor
+    let topping: String?
+    let size: Size
+    
+    func eat() {
+        print("Mmm! I love \(flavor.name)!")
+    }
+}
+
+class IceCreamShop{
+    var flavors: [Cone]
+    var sizes: [Cone]
+    var toppings: [Cone]
+    var totalSales = 0.00
+    
+    init(totalSales: Double) {
+        self.totalSales = totalSales
+        self.flavors = []
+        self.sizes = []
+        self.toppings = []
+    }
+}
+
+
+
+let flavor1 = Flavor(name: "Chocolate", rating: 5)
+let flavor2 = Flavor(name: "Vanilla", rating: 5)
+let flavor3 = Flavor(name: "Coconut", rating: 1)
+let flavor4 = Flavor(name: "Mint", rating: 3)
+let flavor5 = Flavor(name: "Rocky Road", rating: 3)
+var completeMenu: [Flavor] = []
+completeMenu.append(flavor1)
+completeMenu.append(flavor2)
+completeMenu.append(flavor3)
+completeMenu.append(flavor4)
+completeMenu.append(flavor5)
+
+
```

Comment:
Your functions should be inside of your Ice Cream Shop Class. Your icreCreamShop should be able to handle all of these functions like we did with the Airport project.